### PR TITLE
Update instraction after installation path changes

### PR DESCRIPTION
**Pull request by hug-dev**
See https://github.com/parallaxsecond/parsec/issues/250 for details. Very early version now just for early feedback.

### CODE REVIEW

File: src/parsec_service/install_parsec_linux.md [hunk 1]
```diff
@@ -12,54 +12,82 @@ can be a space or comma-separated subset of: `mbed-crypto-provider`, `pkcs11-pro
...
 
-Copy and adapt the [configuration](configuration.md) you want to use. For a secure deployment, make
-sure to activate the `log_error_details` option and to use a `trace` log level.
+Copy and adapt the [configuration](configuration.md) you want to use. Make sure to replace the
+default `store_path` location to:
+
+```````
+-store_path = "./mappings"
++store_path = "/var/lib/parsec/mappings"
+```````
```
Comment (ionut-arm) [inline src/parsec_service/install_parsec_linux.md:87]: I think we should just change that default value.

File: src/parsec_service/install_parsec_linux.md [hunk 2]
```diff
@@ -12,54 +12,82 @@ can be a space or comma-separated subset of: `mbed-crypto-provider`, `pkcs11-pro
...
+system, it will need to be [installed](https://www.rust-lang.org/tools/install) for that specific
+user.
+
+Below is an example with Parsec 0.5.0, update with the version you want!
 
 ```
-cargo install parsec-service --features $DESIRED_FEATURES
+git clone --branch 0.5.0 https://github.com/parallaxsecond/parsec
+cargo build --manifest-path parsec --features $DESIRED_FEATURES --release
+cp parsec/target/release/parsec /usr/libexec/parsec
```
Comment (ionut-arm) [inline src/parsec_service/install_parsec_linux.md:78]: Could we suggest here using `caro install parsec-service --features $DESIRED_FEATURES --path /usr/libexec/parsec`?

**Review by ionut-arm — COMMENTED** [review]
🙆‍♂️ thanks!

File: src/parsec_service/install_parsec_linux.md (hunk 1 above)
Comment (hug-dev) [reply src/parsec_service/install_parsec_linux.md:87]: Sure sounds good!

File: src/parsec_service/build_run.md [hunk 3]
```diff
@@ -20,43 +20,41 @@ replaced by a subset of the features mentioned above, space or comma separated.
...
+The `WARN` log warns us that we are not following the secure installation practices which is fine as
+this is for testing.
 
 From another terminal, it is now possible to execute the [end-to-end tests](tests#end-to-end-tests)
 on Parsec!
 
 ```````
 cd e2e_tests
-cargo test normal_tests
+cargo test --features mbed-crypto-provider normal_tests
```
Comment (ionut-arm) [inline src/parsec_service/build_run.md:57]: Cheers 😅

**Review by ionut-arm — APPROVED** [review]
See comments below, but I'm ok with the changes. 

I'm curious about `/etc/parsec` having `700` permissions - all the `/etc` subdirectories on my WSL have at least read or execute for everyone else 😬

**Comment by hug-dev** [comment]
> I'm curious about /etc/parsec having 700 permissions - all the /etc subdirectories on my WSL have at least read or execute for everyone else 😬

That's true it's similar on my Fedora system. I applied the principle of least privilege, as in I don't think users other than `parsec` need to know what is inside that file (especially if you have confidential information contained in the configuration file name itself). But if it's not Unix-idiomatic I am happt to change, now or in the future.

File: src/parsec_service/install_parsec_linux.md [hunk 4]
```diff
@@ -12,19 +12,50 @@ can be a space or comma-separated subset of: `mbed-crypto-provider`, `pkcs11-pro
...
+Create the `parsec` user.
+
+```
+sudo useradd -m parsec
+sudo passwd parsec
+```
+
+Create the following Parsec directories, with good permissions.
+
+`/var/lib/parsec` for storing persistent data like the `mappings` folder. The service will run from
```
[the remark this answers is not in the document]
Comment (hug-dev) [reply src/parsec_service/install_parsec_linux.md:26]: Yes good point! Will change.

**Comment by ionut-arm** [comment]
> > I'm curious about /etc/parsec having 700 permissions - all the /etc subdirectories on my WSL have at least read or execute for everyone else 😬
> 
> That's true it's similar on my Fedora system. I applied the principle of least privilege, as in I don't think users other than `parsec` need to know what is inside that file (especially if you have confidential information contained in the configuration file name itself). But if it's not Unix-idiomatic I am happt to change, now or in the future.

I'd rather be secure than idiomatic, especially since `config.toml` will contain some pretty sensitive stuff. If someone thinks we're doing things wrong, we can reconsider it

**Review by ionut-arm — APPROVED** [review]

**Pull request merged**